qp/pdf.py

```diff
@@ -55,18 +60,54 @@ def quantize(self, percent=10.0, number=None):
         print("Result: ", self.quantiles)
         return self.quantiles
 
-    def interpolate(self):
+    def interpolate(self, number=100):
+        """
+        Interpolates the quantiles.
 
-        return
+        Parameters
+        ----------
+        number: int
+            The number of points over which to interpolate, bounded by the quantile value endpoints
+
+        Returns
+        -------
+        grid: ndarray, float
+            The input grid upon which to interpolate
+        interpolated : ndarray, float
+            The interpolated points.
+
+        Notes
+        -----
```

Comment:
I added an example to the notes for the approximate() function, though the formatting might be off.